berkeley-mobile/Resources/ResourcesDataSource/ResourceType.swift

```diff
@@ -12,19 +12,13 @@ import UIKit
 enum ResourceType: String {
 
     case health = "Health"
-    case finances = "Finances"
-    case legal = "Legal"
     case basicNeeds = "Basic Needs"
     case admin = "Admin"
```

Comment:
I'd consider adding an unknown/other category in case of future changes. that way, everything backend has (including any new resources) will show up even if not categorized correctly.